packages/snscrape/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=snscrape
+pkgver=v0.4.3.20220106.r73.gcbdaee6
+pkgrel=1
+pkgdesc='A social networking service scraper in Python.'
+arch=('any')
+groups=('blackarch' 'blackarch-recon' 'blackarch-social')
+url='https://github.com/JustAnotherArchivist/snscrape'
+license=('GPL3')
+depends=('libxml2' 'libxslt' 'python' 'python-lxml')
+makedepends=('git')
+source=("git+https://github.com/JustAnotherArchivist/$pkgname.git")
```

Comment:
https://pypi.org/project/snscrape/

use this template instead https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-setuptools

Reply:
Just curiosity @noraj : why do you prefer to retrieve the file from pypi instead of GitHub project repo?

Reply:
PS: used the python-setuptools and implemented the necessary dependencies.

Reply:
> Just curiosity @noraj : why do you prefer to retrieve the file from pypi instead of GitHub project repo?

Because we can keep integrity check, we can check for dependency changes at each release, and we made a script to auto-bump on new release for pypi releases cf. https://github.com/BlackArch/blackarch/blob/master/scripts/up-libraries

Reply:
Thanks